blog/llm-finetuning-2/deepspeed.yaml

```diff
@@ -0,0 +1,36 @@
+name: mistral deepspeed easy
+debug: false
+environment:
+  environment_variables:
+    - NCCL_DEBUG=INFO
+  image: determinedai/genai-train:latest
```

Comment:
let's use determined image here. this one contains genai codebase. we will need to add deepspeed to the requirements.